<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,127);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des traitement</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>771</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0,0,127);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un traitement</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>591</width>
          <height>461</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,127);</string>
        </property>
        <property name="title">
         <string>Ajouter un dossier</string>
        </property>
        <widget class="QLineEdit" name="nom">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>140</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom  de traitement</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>190</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de traitement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="prenom">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>190</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>100</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de traitement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nump">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>100</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>290</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="pic_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>771</width>
           <height>521</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pic_1</zorder>
        <zorder>nom</zorder>
        <zorder>label_16</zorder>
        <zorder>label_21</zorder>
        <zorder>prenom</zorder>
        <zorder>label_22</zorder>
        <zorder>nump</zorder>
        <zorder>pb_ajouter</zorder>
       </widget>
       <widget class="QLabel" name="p1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p1</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les traitement</string>
       </attribute>
       <widget class="QTableView" name="tabpatient">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>731</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="p2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>450</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
       <zorder>p2</zorder>
       <zorder>tabpatient</zorder>
       <zorder>pb_supprimer_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer un traitement</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du traitement</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="p3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p3</zorder>
       <zorder>lineEdit_id_2</zorder>
       <zorder>label_4</zorder>
       <zorder>pb_supprimer</zorder>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Modifier un traitement</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>591</width>
          <height>481</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier un dossier</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de traitement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m1">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>120</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="modif">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>340</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Confirmer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m3">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>240</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_29">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>190</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom  de traitement</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>240</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de traitement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m2">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>190</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="p4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p4</zorder>
       <zorder>groupBox</zorder>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Rechercher un traitement</string>
       </attribute>
       <widget class="QTableView" name="tabr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>741</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du traitement</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="p5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p5</zorder>
       <zorder>tabr</zorder>
       <zorder>label_5</zorder>
       <zorder>rech</zorder>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Rechercher un ordonnance</string>
       </attribute>
       <widget class="QLineEdit" name="rech2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tabr2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>741</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du ordonnance</string>
        </property>
       </widget>
       <widget class="QLabel" name="p6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p6</zorder>
       <zorder>rech2</zorder>
       <zorder>tabr2</zorder>
       <zorder>label_7</zorder>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Rechercher medicament</string>
       </attribute>
       <widget class="QTableView" name="tabord">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>741</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ord">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_horaire">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>150</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Réference medicament</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom produit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nombre de médicaments</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="modif_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>160</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
